src/gui/qgsmanageconnectionsdialog.cpp

```diff
@@ -1226,6 +1272,85 @@ void QgsManageConnectionsDialog::loadGeonodeConnections( const QDomDocument &doc
   }
 }
 
+void QgsManageConnectionsDialog::loadXyzTilesConnections( const QDomDocument &doc, const QStringList &items )
+{
+  QDomElement root = doc.documentElement();
+  if ( root.tagName() != QLatin1String( "qgsXYZTilesConnections" ) )
+  {
+    QMessageBox::information( this, tr( "Loading Connections" ),
```

Comment:
Probably should be QMessageBox::warning here, not information

Reply:
All connection types have that type of message.
Could we change it in a different patch?